app/src/main/java/pl/org/akai/audioai/screens/LoginScreen.kt

```diff
@@ -1,31 +1,147 @@
 package pl.org.akai.audioai.screens
 
-import androidx.compose.foundation.layout.Arrangement
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.Spacer
-import androidx.compose.foundation.layout.height
+import android.graphics.fonts.FontFamily
+import android.widget.Toast
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.magnifier
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.TextField
 import androidx.compose.material3.Button
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.text.style.TextDecoration
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import androidx.navigation.NavController
 import pl.org.akai.audioai.navigation.Screen
 
 @Composable
 fun LoginScreen(navController: NavController) {
 
-    Column (verticalArrangement = Arrangement.Center) {
-        Text("LoginScreen")
-        Spacer(modifier = Modifier.height(8.dp))
-        Button(onClick = {
-            navController.navigate(Screen.AssistantScreen.route)
-        },
-            modifier = Modifier.align(Alignment.End)
-        ) {
-            Text(text = "To AssistantScreen")
+    // Chyba button Login będzie odpowiadał za navi do AssistantScreena
+//    Column(
+//        modifier = Modifier
+//            .padding(20.dp)
+//            .fillMaxHeight(),
+//        verticalArrangement = Arrangement.Bottom,
+//        horizontalAlignment = Alignment.CenterHorizontally
+//    ) {
+//        Spacer(modifier = Modifier.height(8.dp))
+//        Button(
+//            onClick = { navController.navigate(Screen.AssistantScreen.route) },
+//            modifier = Modifier.align(Alignment.End)
+//        ) {
+//            Text(text = "To AssistantScreen")
+//        }
+//    }
+
+    Column(
+        modifier = Modifier
+            .padding(50.dp),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+
+        val context = LocalContext.current
+        val username = remember { mutableStateOf("") }
```

Comment:
```suggestion
        var username by remember { mutableStateOf("") }
```
suggestion: it would be a little bit cleaner to use a delegate here. It allows to set and read the variable without having to call `value` everytime